deploy/grafana/grafana-with-grafana-operator/grafana-cr-template.yaml

```diff
@@ -1,14 +1,29 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-env-vars
+data:
+  GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS: vertamedia-clickhouse-datasource
```

Comment:
this change allow update vertamedia-clickhouse-datasource plugin to 2.1.x version